.pre-commit-config.yaml

```diff
@@ -154,61 +154,21 @@ repos:
       # NOTE: CI automations run both checks. Whenever the versions are
       # NOTE: updated in this config, a symmetric action must be performed
       # NOTE: in `tox.ini` as well.
-      - &_base-mypy-setup-map
-        id: mypy
+      - id: mypy
         alias: mypy-py310
         name: MyPy, for Python 3.10
-        additional_dependencies:
-          - lxml == 4.6.4 # needed for `--txt-report`
+        additional_dependencies: &_mypy_deps
+          - types-dataclasses # Needed for checking py3.6 with a later interpreter
           - types-docutils
           - types-PyYAML
         args:
-          # FIXME: get rid of missing imports ignore
-          - --ignore-missing-imports
-          - --install-types
-          - --namespace-packages
-          - --non-interactive
-          - --pretty
           - --python-version=3.10
-          - --show-column-numbers
-          - --show-error-codes
-          - --show-error-context
-          #- --strict
-          #- --strict-optional
-          - --txt-report
-          - .tox/.tmp/.mypy/
-          - docs/
-          - share/
-          - src/
-          - tests/
         pass_filenames: false
-      - <<: *_base-mypy-setup-map
-        alias: mypy-py36
+      - id: mypy
         name: MyPy, for Python 3.6
-        additional_dependencies:
-          - lxml == 4.6.4 # needed for `--txt-report`
-          - types-dataclasses
-          - types-docutils
-          - types-PyYAML
+        additional_dependencies: *_mypy_deps
         args:
-          # FIXME: get rid of missing imports ignore
-          - --ignore-missing-imports
-          - --install-types
-          - --namespace-packages
-          - --non-interactive
-          - --pretty
           - --python-version=3.6
-          - --show-column-numbers
-          - --show-error-codes
-          - --show-error-context
-          #- --strict
-          #- --strict-optional
-          - --txt-report
```

Comment:
Any motivation for dropping this?

Reply:
Will not in the description, I don't believe anyone was using it and it pulled in a versioned dep

Reply:
Then maybe do this separately? The PR title says "move the config" after all.

Reply:
ok, sure.